src/java/org/apache/cassandra/db/virtual/VirtualKeyspaceRegistry.java

```diff
@@ -73,6 +76,11 @@ public TableMetadata getTableMetadataNullable(TableId id)
         return null != table ? table.metadata() : null;
     }
 
+    public ImmutableSet<String> getKeyspaces()
```

Comment:
How about having a `isVirtualKeyspace(String)` method instead? It looks like we only expose this method to check whether this keyspace is a virtual keyspace or not.

Reply:
Yeah, I think the `exists(keyspace)` semantics makes more sense. I'd prefer if we avoid having to return a copy of the keyspace name set just for the purpose of checking its existence.